tests/unit/scripting.tcl

```diff
@@ -152,7 +152,7 @@ start_server {tags {"scripting"}} {
             local foo = redis.pcall('incr',KEYS[1])
             return {type(foo),foo['err']}
         } 1 mykey
-    } {table {ERR value is not an integer or out of range}}
+    } {table {value is not an integer or out of range}}
```

Comment:
maybe for backwards compatibility we wanna keep the error code in both the string and the code?
this is what caused this change here, right?

Reply:
Yer, please see comment here: https://github.com/redis/redis/pull/10329/files/144c307cff9960de080251b4f86deda999d0d8c6#r812369122